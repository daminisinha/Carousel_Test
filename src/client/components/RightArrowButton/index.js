import React, { Component } from 'react';
import './index.scss'
export default class RightArrow extends Component {

  render() {
    return (
       
        <img onClick={this.props.nextImage} alt="Left Arrow" width="20" height="20px" className="right-arrow float-right" height="60%" src="./images/arrow.svg" alt="Right Arrow" />
 
    );
  }
}

