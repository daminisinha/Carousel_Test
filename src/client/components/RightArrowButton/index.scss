 .right-arrow {
    max-width: 100%;
    height: auto;
    cursor: pointer;
    position: absolute;

    background-color:white;
    top: 30%;
    z-index:999;
    margin-left:-40px;
    float:right;
    border-top-left-radius: 50px;
    border-bottom-left-radius: 50px;
    padding:10px;

    background-color:whitesmoke;
    background-size:30px;
    z-index: 999;
    opacity: 0.5;
    filter: grayscale(100%);
} 

