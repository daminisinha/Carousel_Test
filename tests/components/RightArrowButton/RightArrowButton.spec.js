import React from 'react';
import RightArrowButton from '../../../src/client/components/RightArrowButton'
import {shallow} from 'enzyme';

test('Right arrow button renders',()=>{
    const component = shallow(<RightArrowButton/>); 
    expect(component).toMatchSnapshot();
})
