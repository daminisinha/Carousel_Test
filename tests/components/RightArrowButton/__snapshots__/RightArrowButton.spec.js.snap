// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Right arrow button renders 1`] = `
<img
  alt="Right Arrow"
  className="carousel__right__arrow float-right"
  height="60%"
  src="./images/arrow.svg"
  width="20"
/>
`;
